src/wovnio/wovnphp/Store.php

```diff
@@ -111,7 +111,7 @@ private function defaultSettings()
             'save_memory_by_sending_wovn_ignore_content' => false,
             'enable_wovn_diagnostics' => false,
             'use_cookie_lang' => false,
-            'wovn_debug_mode' => false
```

Comment:
@aw-leigh I think this param should be `debug_mode`, right?

Reply:
Yeah that's probably right.